src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -3602,12 +3602,7 @@ void emitter::emitInsSve_R_R_R(instruction     ins,
             fmt = IF_SVE_GW_3A;
             break;
 
-        case INS_sve_clz:
```

Comment:
The asserts for these instructions were wrong - it allowed cases that don't exist.